_posts/2023-10-20-anvio-org-tutorial.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: blog
```

Comment:
Should this not be layout post? I guess it doesn't matter.

Reply:
we have a layout called `post`? :p

The only ones I saw in `_layouts/` were `blog.html` and `page.html`, and blog seemed more appropriate (plus all the other posts have `layout:blog`)

but if I am missing something, I will change it :)

Reply:
Ah. I see. Anything that is not under _posts should be a page in theory. But I guess it doesn't really matter that much :)

Reply:
then I guess this is good to go?

Reply:
Absolutely! It has always been! :)